styles/variables-base.less

```diff
@@ -195,13 +195,13 @@
 // ---------------------------------------
 @agate-contextual-popup-arrow-height:        30px;
 @agate-contextual-popup-arrow-width:         30px;
-@agate-contextual-popup-border-radius:       inherit;
+@agate-contextual-popup-border-radius:       36px;
 @agate-contextual-popup-font-size:           inherit;
 @agate-contextual-popup-font-weight:         inherit;
 @agate-contextual-popup-offset-overlap:      18px;
 @agate-contextual-popup-offset-small:        9px;
-@agate-contextual-popup-padding-horizontal:  inherit;
-@agate-contextual-popup-padding-vertical:    inherit;
+@agate-contextual-popup-padding-horizontal:  30px;
+@agate-contextual-popup-padding-vertical:    18px;
```

Comment:
maybe it would be a good idea to remove these 3 variables from gallium if they have in variables-base the same value.

Reply:
👍